docs/upgrading.md

```diff
@@ -1,9 +1,10 @@
-> [!NOTE]
-> The official SDK package was renamed from `pinecone-client` to `pinecone` beginning in version 5.1.0.
-> Please remove `pinecone-client` from your project dependencies and add `pinecone` instead to get
-> the latest updates.
```

Comment:
We're using something called `myst-parser` to suck in some content that already existed in the repo as markdown files and render it into the sphinx static site. For the most part it does a good job, but the myst-parser didn't handle this comment block syntax. I think this is a github-specific addition to the markdown syntax, so I guess I'm not surprised.